### ISSUE TITLE
KeyError: 'AVG_MUT_RATE' when specifying mutModel

### ISSUE THREAD

**Issue by aaronkwc** [ncsa/NEAT#14]
**Describe the bug**
Using gen_reads.py with -m option and one of the given models gives error.

Traceback (most recent call last):
  File "/home/aaronkwc/neat-genreads/gen_reads.py", line 901, in <module>
    main()
  File "/home/aaronkwc/neat-genreads/gen_reads.py", line 195, in main
    mut_model = parse_input_mutation_model(mut_model, 1)
  File "/home/aaronkwc/neat-genreads/source/SequenceContainer.py", line 1076, in parse_input_mutation_model
    out_model[0] = pickle_dict['AVG_MUT_RATE']
KeyError: 'AVG_MUT_RATE'

**To Reproduce**
 python ~/neat-genreads/gen_reads.py -r hg19_chrM.fasta -R 101 --pe 300 30 -c 20 -m ~/neat-genreads/models/MutModel_CLLE-ES_ICGC.p -o output_tumor

Tried some other mutModels in the repos but still the same error. 

**Expected behavior**
Not giving an error message

**Comment by joshfactorial** [comment]
We had some broken code in Master. Try checking out tagged version 3.0 (git checkout 3.0), and see if that solves the problem.

**Comment by joshfactorial** [comment]
Okay, I did uncover an issue here. Basically, it had to do with the model files being created in Python2's version of pickle. It was changing the keys from strings to bytes. I went back in and converted all the existing model files to the correct format. I also made the models output as gzipped by default and to assume gzip when reading them in. I also changed the file extensions from .p to .pickle and adjusted the code to fit that scheme. Pulling the latest from master and using the new file names, and let me know if that fixes the problem.
